lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -124,9 +127,14 @@ export default function FilePickerApp({
   const selectURL = url => {
     cancelDialog();
     setUrl(url);
-    submit(true);
+    //submit(true);
```

Comment:
No logic here, just not send it on manual URL assignments and do send it on group set selection